File: DotNet/Messaging/Storage/ServerDistanceInfo.cs

namespace Bollywell.Hydra.Messaging.Storage
{
    public interface IServerDistanceInfo
    {
        string Name { get; set; }
        bool IsReachable { get; set; }
        long Distance { get; set; }
    }

    public class ServerDistanceInfo : IServerDistanceInfo
    {
        public string Name { get; set; }
        public bool IsReachable { get; set; }
        public long Distance { get; set; }
    }
}

File: Java/hydra/src/main/java/uk/co/shastra/hydra/messaging/storage/ServerDistanceInfo.java
package uk.co.shastra.hydra.messaging.storage;

public class ServerDistanceInfo {
	public String name;
	public boolean isReachable;
	public long distance;
	
	public ServerDistanceInfo(String name, boolean isReachable, long distance) {
		super();
		this.name = name;
		this.isReachable = isReachable;
		this.distance = distance;
	}
	
}
